.github/skills/indicator-series/references/decision-tree.md

```diff
@@ -0,0 +1,75 @@
+# Series indicator decision tree
+
+Use this reference to guide Series indicator implementation decisions.
+
+## When to create a new Series indicator
+
+1. **New indicator algorithm**: Implementing a recognized technical analysis indicator
+2. **Canonical reference**: Series provides the mathematical truth for other styles
+3. **Batch processing**: Indicator processes complete datasets efficiently
+
+## Input/output classification
+
+| Input Type | Output Type | Interface | Examples |
+| ---------- | ----------- | --------- | -------- |
+| `IQuote` | Single value | `IReusable` | EMA, SMA, RSI |
+| `IQuote` | Multi-value | `ISeries` | MACD, Alligator |
+| `IReusable` | Single value | `IReusable` | Chained indicators |
+
+## Result model selection
+
+**Single chainable output** → Implement `IReusable`:
+
+```csharp
+public record EmaResult : IReusable
+{
+    public DateTime Timestamp { get; init; }
+    public double? Ema { get; init; }
+    public double Value => Ema.Null2NaN();
+}
+```
+
+**Multiple outputs** → Implement `ISeries`:
```

Comment:
All of these have multiple outputs, only `IReusable` has a defined chainable `Value`